packages/admin-ui/src/Avatar/Avatar.tsx

```diff
@@ -40,13 +40,13 @@ interface AvatarProps
     fallback?: React.ReactElement<AvatarFallbackProps>;
 }
 
-const avatarVariants = cva("relative flex shrink-0 overflow-hidden border-sm border-transparent", {
+const avatarVariants = cva("relative flex shrink-0 overflow-hidden", {
     variants: {
         size: {
-            sm: "text-h6 rounded-sm p-[calc(theme(padding.xs)-theme(borderWidth.sm))] [&>*]:size-md",
-            md: "text-h6 rounded-md p-[calc(theme(padding.xs)-theme(borderWidth.sm))] [&>*]:size-lg",
-            lg: "text-h6 rounded-md p-[calc(theme(padding.sm)-theme(borderWidth.sm))] [&>*]:size-lg",
-            xl: "text-h4 rounded-lg p-[calc(theme(padding.sm)-theme(borderWidth.sm))] [&>*]:size-xl"
+            sm: "text-h6 rounded-sm size-[24px] [&_svg]:size-[16px]",
```

Comment:
Can we utilize [spacing tokens](https://github.com/webiny/webiny-js/blob/9707cc256d258ec3195352e4a95f26877dd07466/packages/admin-ui/src/theme.scss#L168-L181) here?

Reply:
I would love to, but we don't have a `40px` token 😢 
Let's leave it as it is for now...what do you think?

Reply:
OK